lib/rubocop/cop/rails/http_positional_arguments.rb

```diff
@@ -16,6 +16,11 @@ module Rails
       #
       #   # good
       #   get :new, params: { user_id: 1 }
+      #
+      #   # good
```

Comment:
I don't this is necessary to include, it's not such a common approach.

Reply:
What about simplified to `get :new, **options`? I think that might be more straightforward?

Reply:
@andyw8 I made the change I suggested, but if you still don't like it I'm happy to remove it!